test/test_prototype_transforms_consistency.py

```diff
@@ -174,9 +178,83 @@ def parametrization(self):
             DEFAULT_MAKE_IMAGES_KWARGS, color_spaces=[features.ColorSpace.RGB, features.ColorSpace.GRAY]
         ),
     ),
+    ConsistencyConfig(
+        prototype_transforms.ConvertImageDtype,
+        legacy_transforms.ConvertImageDtype,
+        [
+            ArgsKwargs(torch.float16),
+            ArgsKwargs(torch.bfloat16),
+            ArgsKwargs(torch.float32),
+            ArgsKwargs(torch.float64),
+            ArgsKwargs(torch.uint8),
+        ],
+        supports_pil=False,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.ToPILImage,
+        legacy_transforms.ToPILImage,
+        [ArgsKwargs()],
+        make_images_kwargs=dict(
+            color_spaces=[
+                features.ColorSpace.GRAY,
+                features.ColorSpace.GRAY_ALPHA,
+                features.ColorSpace.RGB,
+                features.ColorSpace.RGB_ALPHA,
+            ],
+            extra_dims=[()],
+        ),
+        supports_pil=False,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.Lambda,
+        legacy_transforms.Lambda,
+        [
+            ArgsKwargs(lambda image: image / 2),
+        ],
+        # Technically, this also supports PIL, but it is overkill to write a function here that supports tensor and PIL
+        # images given that the transform does nothing but call it anyway.
+        supports_pil=False,
+    ),
 ]
 
 
+def test_automatic_coverage_deterministic():
+    legacy = {
+        name
+        for name, obj in legacy_transforms.__dict__.items()
+        if not name.startswith("_")
+        and isinstance(obj, type)
+        and not issubclass(obj, enum.Enum)
+        and name
+        not in {
+            "Compose",
+            # This framework is based on the assumption that the input image can always be a tensor and optionally a
+            # PIL image. The transforms below require a non-tensor input and thus have to be tested manually.
+            "PILToTensor",
+            "ToTensor",
+        }
+    }
+    # filter out random transformations
+    legacy = {name for name in legacy if "Random" not in name} - {
+        "AugMix",
+        "TrivialAugmentWide",
+        "GaussianBlur",
+        "RandAugment",
+        "AutoAugment",
+        "ColorJitter",
+        "ElasticTransform",
+    }
+
+    prototype = {config.legacy_cls.__name__ for config in CONSISTENCY_CONFIGS}
```

Comment:
Shouldn't this be ?
```suggestion
    prototype = {config.prototype_cls.__name__ for config in CONSISTENCY_CONFIGS}
```

Reply:
Nope. In some cases we just aliased some prototype transforms to their "legacy" name, for example 

https://github.com/pytorch/vision/blob/97e0ea9c6ebc454538b3fa505e1d199547b0feed/torchvision/prototype/transforms/_type_conversion.py#L60

```py
>>> from torchvision.prototype.transforms import ToPILImage
>>> ToPILImage.__name__
'ToImagePIL'
```

If we now check against the names of the transforms in the `torchvision.transforms` namespace, we get a naming mismatch. By also using the name of the legacy transform here, this goes away.